src/service/repo/signedMessageRepo.ts

```diff
@@ -137,14 +161,29 @@ async function createSignedChat(members: string[]) {
   }
 }
 
+async function deleteMessage(messageId: string) {
+  try {
+    const response = await api.delete(baseUrl.deleteMessage(messageId));
+    if (response.status === 200) {
+      return Promise.resolve(response.data);
+    }
+
+    return Promise.reject(response.data?.status);
+  } catch (e) {
+    console.log(e);
+    return Promise.reject(e);
+  }
+}
+
 const SignedMessageRepo: SignedMessageRepoTypes = {
   checkIsTargetAllowingAnonDM,
   sendSignedMessage,
   getAllSignedChannels,
   getAllSignedPostNotifications,
   getSingleSignedPostNotifications,
   setChannelAsRead,
-  createSignedChat
+  createSignedChat,
+  deleteMessage
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `SignedMessageRepo` object has been updated to include the new methods. This is a good practice as it provides a clear interface for interacting with the repository.


<!-- This is an auto-generated comment by OSS CodeRabbit -->